integration/localnet/bootstrap.go

```diff
@@ -271,6 +271,12 @@ func prepareService(container testnet.ContainerConfig, i int) Service {
 		Environment: []string{
 			"JAEGER_AGENT_HOST=jaeger",
 			"JAEGER_AGENT_PORT=6831",
+			// NOTE: these env vars are not set by default, but can be set [1] to enable binstat logging:
+			// [1] https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers
+			"BINSTAT_ENABLE",
```

Comment:
the note says it is not set by default but isn't it putting them here will automatically include them into containers?

Reply:
The link [1] included in the comment explains how this works in docker and points to another link [2] which says:

> Environment variables with only a key are resolved to their values on the machine Compose is running on, which can be helpful for secret or host-specific values.

The idea is that because `"BINSTAT"` is only a key and has no `=` and value, then it is only created if the docker compose host has that key created.

However, you are successfully making me paranoid and the docker documentation is sufficiently wishy-washy then I'm going to test this again and ensure it works... :-)

[1] https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers
[2] https://docs.docker.com/compose/compose-file/compose-file-v3/#environment

Reply:
@ramtinms I tested a load test with and without `binstat` enabled (please see my notes here [1]), and by default it is disabled which means the `binstat` functions do get called but just return, i.e. minimum overhead.

[1] https://www.notion.so/dapperlabs/Continued-2-e4519b82dfb54fc8b78ca9374eeaf49c